packages/lib/src/interactions/AxialSelectToZoom.tsx

```diff
@@ -73,7 +71,7 @@ function AxialSelectToZoom(props: Props) {
   return (
     <SelectionTool
       onSelectionEnd={onSelectionEnd}
-      id={`AxialSelectToZoom${axis}`}
+      id={`${axis.toUpperCase()}SelectToZoom`}
```

Comment:
Making the ID format more consistent with the name of the prop in `DefaultInteractions` (especially since we may end up allowing people to override interaction IDs).